src/components/Filters/Filters.tsx

```diff
@@ -187,7 +187,7 @@ class FiltersInner extends Component<CombinedProps, State> {
             aria-expanded={filterIsOpen}
           >
             <div className={styles.FilterTriggerLabelContainer}>
-              <h2 className={styles.FilterTriggerTitle}>
+              <h3 className={styles.FilterTriggerTitle}>
```

Comment:
Are they always going to follow a h2? Is this known?

Reply:
They should. The filters content opens into a sheet (portal), so app adjacent. If we make the sheet title an `h2`. Making the filter headers `h3` should remain semantic 👌

Reply:
Interesting, I wonder if there is an example where they would be used in another context. That makes sense though.

Reply:
Nope! The filters components render all of the markup, so unless it changes in Polaris that'll always be the case 😄